packages/sdk/wallets/wallet-hsm-azure/src/azure-key-vault-client.ts

```diff
@@ -13,6 +13,19 @@ import {
 } from '@celo/wallet-hsm'
 import { BigNumber } from 'bignumber.js'
 
+type SupportedCurve = 'P-256K' | 'SECP256K1'
+type SigningAlgorithm = 'ECDSA256' | 'ES256K'
+
+const SUPPORTED_CURVES: readonly SupportedCurve[] = ['P-256K', 'SECP256K1'] as const
+const SIGNING_ALGORITHM_FOR: Record<SupportedCurve, SigningAlgorithm> = {
+  'P-256K': 'ES256K',
+  SECP256K1: 'ECDSA256',
```

Comment:
```suggestion
  'SECP256K1': 'ECDSA256',
```

Reply:
@codyborn I'd like to but the auto-linter disagrees with your sense of style 😂

Reply:
Haha is the `'P-256k'` valid due to the dash?